### PR TITLE
feat: add comments on PRs, regarding code coverage

### PR DESCRIPTION
**Pull request by ssbarbee**

### CODE REVIEW

**Comment by arturbeg** [comment]
<p>Total Coverage: <code>0 %</code></p>
<details><summary>Coverage report</summary>

$ yarn test --coverage
$ jest --silent --coverage

</details>

**Comment by arturbeg** [comment]
<p>Total Coverage: <code>0 %</code></p>
<details><summary>Coverage report</summary>



</details>

**Comment by arturbeg** [comment]
<p>Total Coverage: <code>0 %</code></p>
<details><summary>Coverage report</summary>



</details>

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Title
```
Comment (Sol3iman) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Very nice

**Review by Sol3iman — APPROVED** [review]

**Pull request merged**